app/src/main/java/com/example/ithaca_transit_android_v2/views/DrawRouteCard.kt

```diff
@@ -51,24 +54,60 @@ class DrawRouteCard(context: Context, attrs: AttributeSet?) : View(context, attr
     var circleY7 = 20f //42
     var circleR7 = 8f
 
+    //Walking only Route Card circles
+    var wCircleX1 = 20f
```

Comment:
if it's possible add a comment that describes what makes the dots different. not so important for 1-3, but esp for circles 4-5 where the offsets are the same.